libnetwork/drivers/overlay/ovmanager/ovmanager.go

```diff
@@ -52,22 +52,19 @@ func Init(dc driverapi.DriverCallback, _ map[string]interface{}) error {
 
 // Register registers a new instance of the overlay driver.
 func Register(r driverapi.Registerer, _ map[string]interface{}) error {
-	var err error
-	d := &driver{
-		networks: networkTable{},
-	}
-
-	d.vxlanIdm, err = idm.New(nil, "vxlan-id", 0, vxlanIDEnd)
-	if err != nil {
-		return fmt.Errorf("failed to initialize vxlan id manager: %v", err)
-	}
-
-	return r.RegisterDriver(networkType, d, driverapi.Capability{
+	return r.RegisterDriver(networkType, new(), driverapi.Capability{
 		DataScope:         datastore.GlobalScope,
 		ConnectivityScope: datastore.GlobalScope,
 	})
 }
 
+func new() *driver {
```

Comment:
This collides with the `new` built-in; perhaps we should pick a different name (`newDriver()` ? I see we removed that from the `_test.go`, so we can pick its spot)

(trying to keep the linters at bay 😅)

Reply:
How's `func create()`?

Reply:
Whoops, I totally forgot that `new` was a builtin!